lua/wire/wireshared.lua

```diff
@@ -999,3 +999,71 @@ function nicenumber.nicetime( n )
 		return "0s"
 	end
 end
+
+------------------------------------------------------
+-- Loads a font into the cache if it doesn't exist
+-- Returns true if font exists, else returns false
+------------------------------------------------------
+local _fontcache = {}
+local _defaultFontSize = 18
```

Comment:
I haven't actually come across any constants named like that yet, I was just guessing based on what the rest of the file was like.

Stuff like that should definately be in the style guide when it becomes a thing.